docs/platforms/javascript/common/deployment-provider-setup/netlify.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Netlify
+sidebar_order: 10
+description: "Learn about running Sentry when deploying to Netlify."
+supported:
+  - javascript.nuxt
+---
+
+<Alert level="warning">
+  Deploying on Netlify is still experimental. Currently, not all Sentry features are supported on the server-side. Only your frontend will be monitored.
+</Alert>
+
+<Alert level="info">
+  Before approaching the steps below, read the general documentation around <PlatformLink to="/deployment-provider-setup/general/">setting up Sentry on a deployment provider</PlatformLink> to get quick general overview of the process.
+  This page only shows Netlify-specific steps.
+</Alert>
+
+## Approach 1: Adding the `--import` flag
+
+<PlatformSection supported={['javascript.nuxt']}>
+  The server-side of Nuxt is deployed as a Netlify Function. So make sure to scope the environment variable `NODE_OPTIONS` to "Functions".
+  The path to the function is depending on your setup. When building the preset locally, the file is available at: `.netlify/functions-internal/server/sentry.server.config.mjs`.
+
+  <Alert level="warning">
+    The docs for this are still vague and we will update them as soon as we know more about which exact path to use for the `--import` flag.
```

Comment:
I guess what you want to say here is that the docs are not really ready/done here yet? "vague" implies for me that everything works but is not documented well 🤔 Maybe rather:

```suggestion
    The docs for this are work in progress and we will update them as soon as we know more about which exact path to use for the `--import` flag.
```

Reply:
Also, does this work as of today? If it does not _actually_ work we should not document it, and/or call out very specifically that it does not work!

Reply:
+1 to @mydea. I think it's best not to document it if we know it doesn't work.